Librarian/Series.js

```diff
@@ -36,21 +33,20 @@ Series.prototype.fetch = (done)=> {
   return;
 };
 
-
 /**
 SeriesコレクションからBookListコレクションを取得する
 **/
-Series.prototype.join = function(seriesItems, done){
-  var conditions = {
+Series.prototype.join = function join(seriesItems, done) {
+  const conditions = {
     _id: {
-      $in: seriesItems.map(function(seriesItem){
+      $in: seriesItems.map((seriesItem)=> {
         return seriesItem._id;
-      })
-    }
+      }),
+    },
   };
 
-  this.BookList.find(conditions, function(err, books){
-    if(err){
+  this.BookList.find(conditions, (err, books)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).